adam/ontology/phase1_ontology.py

```diff
@@ -528,7 +534,8 @@ def subtype(sub: OntologyNode, _super: OntologyNode) -> None:
 subtype(_FOOT, _BODY_PART)
 
 # Verbs
-
+WALK = OntologyNode("walk")
+subtype(WALK, ACTION)
```

Comment:
You'll need to define the `WALK` action similarly to how the other actions are defined further down in the ontology file -- I'd suspect a test to generate the walk action should fail currently.

Reply:
defined this later in the file similarly to `COME` since both indicate motion to a goal, but I can update it if that's not correct.

Reply:
Following @gabbard's guidance from slack let's remove the goal from the action and add in an `aux_surface` similar to roll.

Reply:
Updated